Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -3468,6 +3469,9 @@ - (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer
 
 #if ENABLE(MODEL_PROCESS)
     if (gestureRecognizer == _modelInteractionPanGestureRecognizer) {
+        if (!_page->hasModelElement())
```

Comment:
It looks like `hasModelElement()` is only used when `ENABLE(MODEL_PROCESS)` is `1` but defined when `ENABLE(MODEL_ELEMENT)` is `1`.

Should it only be defined  when `ENABLE(MODEL_PROCESS)` is `1`?